experiments/normalscan.py

```diff
@@ -9,52 +17,18 @@ def __init__(self):
         cpid = 150
         super(Normalscan, self).__init__(cpid)
 
-        tx_antennas = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15]
-        rx_main_antennas = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15]
-        rx_int_antennas = [0, 1, 2, 3]
-        pulse_sequence = [0, 9, 12, 20, 22, 26, 27] #[0, 14, 22, 24, 27, 31, 42, 43]
-        tau_spacing = 2400 # 1500 # us
         self.add_slice({  # slice_id = 0, there is only one slice.
-            "tx_antennas": tx_antennas,
-            "rx_main_antennas": rx_main_antennas,
-            "rx_int_antennas": rx_int_antennas,
-            "pulse_sequence": pulse_sequence,
-            "pulse_phase_offset": [0, 0, 0, 0, 0, 0, 0],
-            "tau_spacing": tau_spacing,  # us
-            "pulse_len": 300,  # us
-            "num_ranges": 75,  # range gates
-            "first_range": 180,  # first range gate, in km
+            "pulse_sequence": scf.SEQUENCE_7P,
+            "tau_spacing": scf.TAU_SPACING_7P,
+            "pulse_len": scf.PULSE_LEN_45KM,
+            "num_ranges": scf.STD_NUM_RANGES,
+            "first_range": scf.STD_FIRST_RANGE,
             "intt": 3500,  # duration of an integration, in ms
-            #"intn": 21,  # number of averages if intt is None.
-            "beam_angle": [-26.25, -22.75, -19.25, -15.75, -12.25, -8.75,
-                           -5.25, -1.75, 1.75, 5.25, 8.75, 12.25, 15.75, 19.25, 22.75,
-                           26.25],
-            "beam_order": [15, 14, 13, 12, 11, 10, 9, 8, 7, 6, 5, 4, 3, 2, 1, 0],
-            #"scanboundflag": True,  # there is a scan boundary
-            #"scanbound": 60000,  # ms
-            "txfreq": 10500,
+            "beam_angle": scf.STD_16_BEAM_ANGLE,
+            "beam_order": scf.STD_16_REVERSE_BEAM_ORDER,
```

Comment:
Beam order will depend upon the radar site, FYI. We will need to have some way of automating this? A 'lookup table' dictionary of the radars with their standard frequencies and beam directions perhaps.

Reply:
This direction is in the hdw.dat files actually, field number 7 tells you, depending upon if it is a negative or positive number, which direction the radar scans.